text/0000-lang-team-mcp.md

```diff
@@ -0,0 +1,216 @@
+
+## Transparency around bandwidth
+
+As part of this process, the lang-team plans to organize and expose more clearly which project groups each member is currently involved with (likely using a Github project to expose details and other information). Similarly, we plan to communicate more clearly the sorts of proposals we are looking for at any period of time via blog posts or information on the [lang-team website].
+
+[lang-team website]: https://lang-team.rust-lang.org/
+
+## Transitioning to the new system
+
+There are a number of existing RFCs, many of which have received quite a lot of work, that have to be transitioned to the new system. We do not want to just "auto close" all of those RFCs because in many cases they represent a significant amount of effort and "near consensus".
+
+Our plan is to first create the new system and encourage folks to create proposals who wish to. Presuming our experience with the new system is positive, and once we have "ironed out the kinks", we will begin a process to port over the existing RFCs. 
+
+The precise plans will be determined then, but they will likely include:
+
+* A review of the RFCs to try and find those lang-team liaisons where possible.
+* Announcing a date when we will complete the migate and posting a notice on existing RFCs. This notice would encourage others to try out the new process. After that date expires, existing RFCs would be closed.
+* An alternative might be to "auto-migrate" all existing proposals via automation to create proposals, perhaps spaced out in groups.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+* There is an additional step to prepare an RFC, which indicates more overhead and procedures to be followed.
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+* We could have proposals be opened on the RFC repository
+    * Why? Central place for all proposals across teams.
+    * Why not? The RFC repository is no longer a "high signal" place to watch for "changes that have traction".
+* We could try this for all teams at once.
+    * Why? Uniform process is good.
+    * Why not? Not sure how it will work, and each team has somewhat different needs.
+* We could have MCPs be opened as a PR
+    * In earlier versions of this RFC, we made the MCP proposas be a PR against the lang-team repo that contained the proposal.
+    * The idea here was that it would be useful to have the text of the proposal as something we can merge as "not accepted" in order to have a better record.
+    * We ultimately moved to an issue for a few reasons:
+        * YAGNI: The tooling and workflow seemed more esoteric. The idea of opening first to a "not accepted" directory and then merging just felt a bit "unexpected" and harder to explain and envision.
+        * It is hard to collaborate actively on a PR; a hackmd or other such document is better for this purpose.
```

Comment:
The implied workflow being using a hackmd prior to opening the MCP, and migrating to an issue when the text is deemed ready?

Reply:
I think my assumption was that -- much like compiler-team MCPs -- the longer text is often included "out of line", in a hackmd or gist.

Reply:
So I've been giving this some thought (and we had some [discussion on Zulip](https://zulip-archive.rust-lang.org/213817tlang/56024proposalsissuesvsPRs.html)) and I think I've come to the conclusion that -- presuming we continue to use issues, which I still lean towards -- we should have people paste the full text of the proposal into the issue.

In this way, the closed issue still serves as a nice record of the proposal.

Reply:
I removed reference to hackmd.